### ISSUE TITLE
example curl that actually works?

### ISSUE THREAD

**Issue by adamgf** [basecamp/bc3-api#316]
Can someone please provide an example curl that actually works from terminal on Mac OS X?

**Comment by adamgf** [comment]
When I try do it through a super simple React app using axios it tells me that I need to provide a header for 'Access-Control-Allow-Origin' which I provided but am still getting that same error message.  This is what I am getting back from the React app:

xhr.js:162 Refused to set unsafe header "User-Agent"
(anonymous) @ xhr.js:162
c @ utils.js:251
(anonymous) @ xhr.js:156
e.exports @ xhr.js:15
e.exports @ dispatchRequest.js:56
c.request @ Axios.js:108
(anonymous) @ bind.js:9
(anonymous) @ App.js:25
c @ runtime.js:63
(anonymous) @ runtime.js:294
(anonymous) @ runtime.js:119
r @ asyncToGenerator.js:3
l @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ App.js:23
(anonymous) @ App.js:28
r @ oauth.min.js:2
fireWith @ oauth.min.js:2
Deferred.T.each.o.<computed> @ oauth.min.js:2
(anonymous) @ oauth.min.js:1
r @ oauth.min.js:2
fireWith @ oauth.min.js:2
n @ oauth.min.js:2
(anonymous) @ oauth.min.js:2
load (async)
send @ oauth.min.js:2
ajax @ oauth.min.js:2
o @ oauth.min.js:1
http_me @ oauth.min.js:1
(anonymous) @ oauth.min.js:1
(anonymous) @ App.js:13
r @ oauth.min.js:2
fireWith @ oauth.min.js:2
Deferred.T.each.o.<computed> @ oauth.min.js:2
(anonymous) @ oauth.min.js:1
(anonymous) @ oauth.min.js:2
r @ oauth.min.js:2
fireWith @ oauth.min.js:2
Deferred.T.each.o.<computed> @ oauth.min.js:2
(anonymous) @ oauth.min.js:1
(anonymous) @ oauth.min.js:2
r @ oauth.min.js:2
fireWith @ oauth.min.js:2
n @ oauth.min.js:2
(anonymous) @ oauth.min.js:2
load (async)
send @ oauth.min.js:2
ajax @ oauth.min.js:2
retrieveMethods @ oauth.min.js:1
sendCallback @ oauth.min.js:1
p @ oauth.min.js:1
adamgf.github.io/:1 Access to XMLHttpRequest at 'https://3.basecampapi.com/23200586/people.json' from origin 'https://adamgf.github.io' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
xhr.js:210 GET https://3.basecampapi.com/23200586/people.json net::ERR_FAILED
(anonymous) @ xhr.js:210
e.exports @ xhr.js:15
e.exports @ dispatchRequest.js:56
c.request @ Axios.js:108
(anonymous) @ bind.js:9
(anonymous) @ App.js:25
c @ runtime.js:63
(anonymous) @ runtime.js:294
(anonymous) @ runtime.js:119
r @ asyncToGenerator.js:3
l @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ App.js:23
(anonymous) @ App.js:28
r @ oauth.min.js:2
fireWith @ oauth.min.js:2
Deferred.T.each.o.<computed> @ oauth.min.js:2
(anonymous) @ oauth.min.js:1
r @ oauth.min.js:2
fireWith @ oauth.min.js:2
n @ oauth.min.js:2
(anonymous) @ oauth.min.js:2
load (async)
send @ oauth.min.js:2
ajax @ oauth.min.js:2
o @ oauth.min.js:1
http_me @ oauth.min.js:1
(anonymous) @ oauth.min.js:1
(anonymous) @ App.js:13
r @ oauth.min.js:2
fireWith @ oauth.min.js:2
Deferred.T.each.o.<computed> @ oauth.min.js:2
(anonymous) @ oauth.min.js:1
(anonymous) @ oauth.min.js:2
r @ oauth.min.js:2
fireWith @ oauth.min.js:2
Deferred.T.each.o.<computed> @ oauth.min.js:2
(anonymous) @ oauth.min.js:1
(anonymous) @ oauth.min.js:2
r @ oauth.min.js:2
fireWith @ oauth.min.js:2
n @ oauth.min.js:2
(anonymous) @ oauth.min.js:2
load (async)
send @ oauth.min.js:2
ajax @ oauth.min.js:2
retrieveMethods @ oauth.min.js:1
sendCallback @ oauth.min.js:1
p @ oauth.min.js:1
createError.js:16 Uncaught (in promise) Error: Network Error
    at e.exports (createError.js:16)
    at XMLHttpRequest.y.onerror (xhr.js:117)

So what headers do I need to use after I Oauth 2.0 in???

**Comment by adamgf** [comment]
And this does not return anything at all?

adam@Adams-Mac-mini sbdprojectmanager % curl -H "Authorization: Bearer BAhbB0kiAbB7ImNsaWVudF9pZCI6ImExZjM1YzdmNWQ5M2JjMWZiYmU4MjcwNDIyYzI2MTgwYzNiMDk2NzgiLCJleHBpcmVzX2F0IjoiMjAyMS0xMS0yMVQyMjo0MTo0MVoiLCJ1c2VyX2lkcyI6WzQ1MTYyMjkyXSwidmVyc2lvbiI6MSwiYXBpX2RlYWRib2x0IjoiMDk2OTcwNzUxNGU1ZjIzYjUwODk4MTZjZmVkNTI2MWIifQY6BkVUSXU6CVRpbWUNtmoewGB1naYJOg1uYW5vX251bWkCDAI6DW5hbm9fZGVuaQY6DXN1Ym1pY3JvIgdSQDoJem9uZUkiCFVUQwY7AEY=--c1d96a536f344e3e399042f31ddbf065e56f865f" \
  -H 'Content-Type: application/json' \
  -H 'User-Agent: SBDProjectManager (adamgf@gmail.com)' \
  -d '{ "name": "My new project!" }' \
  https://3.basecampapi.com/23200586/projects.json

**Comment by adamgf** [comment]
I was not using the correct user id.  I was grabbing numbers from the response and I grabbed the wrong number!  I think it was in result.me().done(function(data) { it was the data.id which I figured was the field that I needed but it wasn't in case somebody runs into the same issue.

**Comment by lewispb** [comment]
Hey @adamgf - I'm glad you got that figured out! Just to let you know I've deleted your comment above containing your bearer token to prevent unauthorized access to your account. I would strongly suggest you change the password on your account to rotate your credentials now too.
